fullscreen/api/active-document-manual.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<title>Element ready check for containing iframe, element's document is not the active document</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../trusted-click.js"></script>
+<div id="log"></div>
+<iframe allowfullscreen></iframe>
+<script>
+async_test((t) => {
+  onload = () => {
```

Comment:
Don't need to wait for load I think, but if you `t.step_func` is needed. (Or is it? I can never remember if testharness.js gets it right if there's just a single async_test.